main/Checkbox/index.tsx

```diff
@@ -111,30 +117,28 @@ const Checkbox: FC<CheckboxProps> = ({
   labelColor,
   onPress,
   item,
-  /* TODO  */
   defaultChecked = false,
-  // disabled = false,
 }) => {
   const isSelected = item.selected || defaultChecked;
-
+  const isDisabled = item.disabled;
   return (
     <TouchableHighlight
       onPress={(): void => onPress?.(item.value)}
       underlayColor="transparent"
-      style={{ marginHorizontal: 20 }}>
+      style={{ marginHorizontal: 20, paddingBottom: 20 }}
+      disabled={isDisabled}>
       <Container>
         <MarkerContainer
           boxSize={boxSize}
-          boxColor={boxColor}
+          boxColor={isDisabled ? COLOR.LIGHTGRAY : boxColor}
         >
           <Marker isSelected={isSelected}>
             {isSelected && <MarkerImg
               source={require('../__assets__/check_tick.png')}
             />}
           </Marker>
         </MarkerContainer>
-        <Label labelSize={labelSize} labelColor={labelColor}>
-          {item.label}
+        <Label labelSize={labelSize} labelColor={isDisabled ? COLOR.LIGHTGRAY : labelColor}>          {item.label}
```

Comment:
Is there a reason for this subspace? I think the existing is better for readability